Sources/UINotificationPresentationContext.swift

```diff
@@ -10,71 +10,71 @@ import UIKit
     public let containerWindow: UIWindow
-    
+
     /// The level the container window should be on when presenting the notification
     private let windowLevel: UIWindow.Level
-    
+
     /// The `UINotificationView` containing the visual representation of the `UINotification`.
     public let notificationView: UINotificationView
-    
+
     /// The notification request currenly being handled.
     private let request: UINotificationRequest
-    
+
     /// Public getter for the current notification which is handled.
     public var notification: UINotification {
         return request.notification
     }
-    
+
     internal init(request: UINotificationRequest, containerWindow: UIWindow, windowLevel: UIWindow.Level, notificationView: UINotificationView) {
         self.request = request
         self.containerWindow = containerWindow
         self.notificationView = notificationView
         self.windowLevel = windowLevel
-        
+
         prepareContainerWindow()
         prepareNotificationView()
     }
-    
+
     /// Completes the presentation. Resets the container window and updates the state of the `UINotificationRequest`.
     public func completePresentation() {
         resetContainerWindow()
         notificationView.removeFromSuperview()
         request.finish()
-        
+
         // This releases all objects.
         // We can't define the presenter weak inside the notificationView, because it's needed for dismissing after a pan gesture.
         notificationView.presenter = nil
     }
-    
+
     private func prepareContainerWindow() {
         containerWindow.windowLevel = windowLevel
         #if !TEST
             containerWindow.isHidden = false
         #endif
     }
-    
+
     private func prepareNotificationView() {
         /// Create a container view controller to let it handle orientation changes.
         let containerViewController = UIViewController(nibName: nil, bundle: nil)
         containerViewController.view.addSubview(notificationView)
         containerWindow.rootViewController = containerViewController
-        
+
         /// Set the top constraint.
         var notificationViewTopConstraint = notificationView.topAnchor.constraint(equalTo: containerViewController.view.topAnchor, constant: -notification.style.height.value)
-        
+
         /// For iPhone X we need to use the safe area layout guide, which is only available in iOS 11 and up.
         if #available(iOS 11.0, *) {
             notificationViewTopConstraint = notificationView.topAnchor.constraint(equalTo: containerViewController.view.safeAreaLayoutGuide.topAnchor, constant: -notification.style.height.value)
         }
-        
+
         notificationView.topConstraint = notificationViewTopConstraint
-        
+
         var constraints = [
-            notificationView.leftAnchor.constraint(equalTo: containerViewController.view.leftAnchor).usingPriority(.defaultHigh),
```

Comment:
It would be better to use `leading` and `trailing` instead of `left`and `right`. Would it make sense to include it in the changes, as it is very small?